src/Npgsql/NpgsqlConnector.cs

```diff
@@ -1328,7 +1328,7 @@ void Cleanup()
             Log.Trace("Cleaning up connector", Id);
             try
             {
-                _stream.Dispose();
+                _stream?.Dispose();
```

Comment:
I seem to remember case(s) of this happening after network errors... It may no longer be relevant, but do we think there's a perf issue with having the try/catch here for that case?